Update README.md (#3)
arribada/SMARTConnect: README.md

```

NOTES:
to trigger a new image build an push with circle, just create a new tag or delete and recreate an existing tag.

## Configuring SMART Desktop to access the Server

Additional steps must now be taken in order for a SMART Desktop installation to be able to access the server.

The certificate to be used when configuring SMART Desktop to access the server is found at [tomcat/ssl.cert](tomcat/ssl.cert)

The certificate is issued to the name `smartconnect` so need to link the SMART connect server IP to that name.

Edit the system host file to link the certificate hostname to the correct IP
C:\Windows\System32\drivers\etc\hosts

> Append the following two lines to the file

Reviewer: ```suggestion
Edit the system host file to link the certificate hostname to the correct IP
`C:\Windows\System32\drivers\etc\hosts`
Append the following two lines to the file
```

> Append the following two lines to the file

Reviewer: ```suggestion
Edit the system host file to link the certificate hostname to the correct IP
```C:\Windows\System32\drivers\etc\hosts```
Append the following two lines to the file
```

> Append the following two lines to the file

Reviewer: ```suggestion
Edit the system host file to link the certificate hostname to the correct IP
```C:\Windows\System32\drivers\etc\hosts```
Append the following two lines to the file

> Append the following two lines to the file

Reviewer: ```suggestion
Edit the system host file to link the certificate hostname to the correct IP

`C:\Windows\System32\drivers\etc\hosts`

Append the following two lines to the file
```

```

NOTES:
to trigger a new image build an push with circle, just create a new tag or delete and recreate an existing tag.

## Configuring SMART Desktop to access the Server

Additional steps must now be taken in order for a SMART Desktop installation to be able to access the server.

> The certificate to be used when configuring SMART Desktop to access the server is found at https://github.com/arribada/SMARTConnect/blob/master/tomcat/ssl.cert

Reviewer: ```suggestion
The certificate to be used when configuring SMART Desktop to access the server is found at [tomcat/ssl.cert](tomcat/ssl.cert)
```
let use relative paths in case we rename something.

> Additionally, in order to use this certificate you must add an alias to your host file so that the IP address of the hosting machine is usable with the certificate.

Reviewer: ```suggestion
The certificate is issued to the name `smartconnect` so need to link the SMART connect server IP to that name.
```

> First go to https://www.whatismyip.com/ on the machine hosting the Connect Server to find the IP address of that machine.

Reviewer: ```suggestion
```
let's not put specific network details as these could vary quite a bit, for example, can access the smart connect server locally, than using the local IP would be preferred, over the internet would need port forwarding, with VPN would need another setup etc. Lets leave out these details as they vary quite a bit.

> Open a Notepad as Administrator and open the file located at C:\Windows\System32\drivers\etc\hosts

Reviewer: ```suggestion
Edit the system host file to link the certificate hostname to the correct IP
C:\Windows\System32\drivers\etc\hosts
```

Append the following two lines to the file

```
#SMARTCONNECT
<IP.Of.Hosting.Machine> smartconnect
```

> At this point you should be able to use the provided certificate file and the smart:smart or customized admin credentials  in Connect to configure a SMART Desktop instance to access the server running locally using the server address: https://smartconnect:8443/server

Reviewer: ```suggestion
At this point, you will be able to connect SMART desktop to `https://smartconnect:8443/server` with the provided certificate file and the default smart:smart login credentials. 
```

> Note: If the IP address of the machine hosting SMART Connect is not static then the alias in the hosts file will need to be updated as the IP of the hosting machine changes.

Reviewer: ```suggestion
```

I think these are network-specific settings and no need to go in such details. In practice there could be many different variations here. Accessing it locally, through firewalls, VPN etc.
